docs/content/docs/connectors/pipeline-connectors/kafka.md

```diff
@@ -202,79 +266,100 @@ An output example is:
 
 Data Type Mapping
 ----------------
+[Literal type](https://debezium.io/documentation/reference/3.1/connectors/mysql.html#mysql-data-types): defines the physical storage format of data (type field of the debezium schema)<br>
+[Semantic type](https://debezium.io/documentation/reference/3.1/connectors/mysql.html#mysql-data-types): defines the logical meaning of data (name field of the debezium schema).
 <div class="wy-table-responsive">
 <table class="colwidths-auto docutils">
     <thead>
       <tr>
         <th class="text-left">CDC type</th>
         <th class="text-left">JSON type</th>
+        <th class="text-left">Literal type</th>
+        <th class="text-left">Semantic type</th>
```

Comment:
Do we need to add Literal/Semantic  type to the documentation? It seems like users won't care about it? @lvyanquan

Reply:
Yes, I suggest not add this as users won't pay much attention on it.